.github/workflows/release-plz.yml

```diff
@@ -10,25 +10,50 @@ on:
       - develop
 
 jobs:
-  release-plz:
-    name: Release-plz
+
+  # Release unpublished packages.
+  release-plz-release:
+    name: Release-plz release
     runs-on: ubuntu-latest
+    if: startsWith(github.event.head_commit.message, "chore: release")
```

Comment:
can you have `if` up here or does it need to live in `steps` ?

Reply:
ah nvm it indeed can live here 👍

Reply:
Rats you were right https://github.com/spiraldb/vortex/actions/runs/11636435105/workflow

Reply:
Ah I am maybe just missing ${{ }}?